### DISCUSSION TITLE
How can I change the 2D part of MVX-Ne t(change FasterRCNN into FCOS)

### DISCUSSION THREAD

**Discussion by zhongqiu1245** [open-mmlab/mmdetection3d#2157] [General]
Could you tell me how can I change the 2D part of MVX-Net(like change FasterRCNN into FCOS)
To accelerate the speed of MXV-Net, I want to change the fasterrcnn of MVX-Net into fcos. But how? Thank you!

**Comment by thelandord** [comment]
hi,bro.Have you already implemented it? I also  want to change the MULTIMODAL .but i don't how to do
